classroom.rb

```diff
@@ -0,0 +1,59 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @students << student
+    student.classroom = self
+  end
+end
+
+class Student
+  attr_accessor :name, :classroom
+
+  def initialize(name)
+    @name = name
+    @classroom = nil
+  end
+end
+
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+  end
+end
+
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+    book.add_rental(self)
+    person.add_rental(self)
+  end
+end
+
+class Person
+  attr_accessor :name
+  attr_reader :rentals
+
+  def initialize(name)
+    @name = name
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals << rental
+  end
+end
```

Comment:
- [ ] for this to work Getters and Setters for @rentals and @age should be present.
- [ ] the method for adding rentals looks different than this. have a look at the link to the example in a previous lesson: https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship